convert2rhel/unit_tests/utils_test.py

```diff
@@ -147,13 +151,26 @@ def test_download_pkg_success_with_all_params(self):
                 "--enablerepo=repo1",
                 "--enablerepo=repo2",
                 "--releasever=8",
+                "--setopt=varsdir=/tmp",
                 "--setopt=module_platform_id=platform:el8",
                 "kernel",
             ],
             utils.run_cmd_in_pty.cmd,
         )
         self.assertTrue(path)  # path is not None (which is the case of unsuccessful download)
 
+    @unit_tests.mock(system_info, "releasever", None)
+    def test_download_pkg_assertion_error(self):
+        with pytest.raises(AssertionError) as e:
+            utils.download_pkg(
+                pkg="kernel",
+                set_releasever=True,
+                custom_releasever=None,
+            )
+
+            assert "custom_releasever or system_info.releasever must be set." == e.value
```

Comment:
Since we're doing `with pytest.raises`. Does it ever reach this check? Isn't it indented one too many?